docs/resources/Guild.md

```diff
@@ -1042,11 +1061,11 @@ Returns a list of [ban](#DOCS_RESOURCES_GUILD/ban-object) objects for the users
 
 ###### Query String Params
 
-| Field     | Type      | Description                                    | Default |
-|-----------|-----------|------------------------------------------------|---------|
-| limit?    | number    | number of users to return (up to maximum 1000) | 1000    |
-| before? * | snowflake | consider only users before given user id       | null    |
-| after? *  | snowflake | consider only users after given user id        | null    |
+| Field      | Type      | Description                                    | Default |
+|------------|-----------|------------------------------------------------|---------|
+| limit?     | number    | number of users to return (up to maximum 1000) | 1000    |
+| before? \* | snowflake | consider only users before given user id       | null    |
+| after? \*  | snowflake | consider only users after given user id        | null    |
 
 \* Provide a user id to `before` and `after` for pagination. Users will always be returned in ascending order by `user.id`. If both `before` and `after` are provided, only `before` is respected.
```

Comment:
```suggestion
\* Provide a user id to `before` or `after` for pagination. Users will always be returned in ascending order by `user.id`. If both `before` and `after` are provided, only `before` is respected.
```

Reply:
that's not in scope of this pr, just fixed the table + marking